integration_tests/src/main/python/spark_session.py

```diff
@@ -71,6 +71,10 @@ def _set_all_confs(conf):
         _spark.conf.set("spark.rapids.sql.test.injectRetryOOM", "true")
     else:
         _spark.conf.set("spark.rapids.sql.test.injectRetryOOM", "false")
+    if disable_timezone_test():
+        _spark.conf.set("spark.sql.session.timeZone", "UTC")
+    else:
+        _spark.conf.set("spark.sql.session.timeZone", "Asia/Shanghai")
```

Comment:
Will remove this block before merge.
After remove this, the test method is:
export TEST_TZ="Iran"
./integration_tests/run_pyspark_from_build.sh